libs/lume/scripts/build/build-release-notarized.sh

```
@@ -123,86 +191,66 @@ else
-  log "normal" "[lume build] Note: $USER_BIN is not in your PATH. Add 'export PATH=\"$USER_BIN:\$PATH\"' to your shell profile."
-fi
+# --- Create release archives ---
 
 # Get architecture and create OS identifier
 ARCH=$(uname -m)
 OS_IDENTIFIER="darwin-${ARCH}"
+RELEASE_DIR="$(cd .release && pwd)"
 
-# Create versioned archives of the package with OS identifier in the name
 log "essential" "Creating archives in $RELEASE_DIR..."
 cd "$RELEASE_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)